particle/architecture/component-structure.md

```diff
@@ -2,7 +2,7 @@
 
 ## Generating a Component
 
-Components have a specific file structure. Instead of making a developer create all required files by hand, we use a [Yeoman](http://yeoman.io/) generator to easily create new component folders. Simply run:
+Components have a specific file structure. Instead of making a developer create all required files by hand, we use a [Yeoman](http://yeoman.io/) generator to easily create new component folders by running the following command in Bash:
```

Comment:
I prefer the original terminology here.

Reply:
I think compromising with `by running the following command:` would be best. Bash can't be relied upon to be the same Terminal everyone's using, and "Simply" is a bit casual.